app/src/main/res/layout/session_layout_land_large.xml

```diff
@@ -37,7 +37,7 @@
             android:layout_marginLeft="5dp"
             android:adjustViewBounds="true"
             android:baseline="14sp"
-            android:src="@drawable/bell" />
```

Comment:
I think there is left-over files here:

- app/src/cccamp2019/res/drawable/bell.png
- app/src/jev2022/res/drawable/bell.png